.github/workflows/push_pr.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: |
           changed=$(ct --config .github/ct.yaml list-changed)
           if [[ -n "$changed" ]]; then
-            echo "::set-output name=changed::true"
+            echo "changed=true" >> $GITHUB_OUTPUT
```

Comment:
Why we are following a different notation? Is it because we did not declare the output like:
```
    outputs:
      version: ${{ steps.version.outputs.next-version }}
      image-tag: ${{ steps.image-tag.outputs.IMAGE_TAG }}
```